process-controller/src/main/java/org/jboss/as/process/ManagedProcess.java

```diff
@@ -107,7 +125,9 @@ enum State {
         this.pcAuthKey = pcAuthKey;
         isPrivileged = privileged;
         respawnPolicy = respawn ? RespawnPolicy.RESPAWN : RespawnPolicy.NONE;
-        log = Logger.getMessageLogger(ProcessLogger.class, "org.jboss.as.process." + processName + ".status");
+        logStatus = Logger.getMessageLogger(ProcessLogger.class, "org.jboss.as.process." + processName + ".status");
+        logSystemErr = Logger.getMessageLogger(ProcessLogger.class, "org.jboss.as.process." + processName + ".system.err");
+        logSystemOut = Logger.getMessageLogger(ProcessLogger.class, "org.jboss.as.process." + processName + ".system.out");
```

Comment:
Not critical, should we use `stdout` and `stderr` instead? I don't have a strong opinion, I'm just curious :)

We also might want to consider the order to have the process name last to make filtering easier. For example you could just skip `stdout` with `logger.org.jboss.as.process.system.out.level=WARN`

Reply:
> Not critical, should we use stdout and stderr instead?

I am fine with both. `xxx.system.out` and `xxx.system.err` are very intuitive to me due to the similitude with the Java methods, if `xxx.system.stdout` and `xxx.system.stderr` are more related to logging world, I am fine with them too. I do not have any preference.
 

> We also might want to consider the order to have the process name last to make filtering easier. For example you could just skip stdout with logger.org.jboss.as.process.system.out.level=WARN

Thanks, that's a smarter choice. I agree with you.

We already have the `.status` category defined as `"org.jboss.as.process." + processName + ".status"`. Should we change it too? 

That would make all the related categories consistent, although would affect existing configurations. However, since are logging categories, any existing configuration can be changed to adapt to the new version and that would not affect server behaviour, so I am in favor of adapting those three.

Reply:
@jamezp 

I've changed the categories to `xxx.system.stdout` and `xxx.system.stderr`.

However, I left the `"org.jboss.as.process." + processName + ".system.sterr"` and `"org.jboss.as.process." + processName + ".system.stout"` as proposed originally since we already have `"org.jboss.as.process." + processName + ".status"` and users could be familiar with that category. 

I prefer being conservative here since this is an enhancement and I don't want to introduce a change on the `.status` category or have categories working differently than the status one. I think the usage of them is unlikely, but if they are needed I think is preferable a consistent with what we previously have.